lib/calculation/process.ak

```diff
@@ -616,13 +629,15 @@ test process_orders_test() {
       0,
       0,
       0,
+      True,
     )
 
   expect new_a == 1_001_000_000
   expect new_b == 1_001_000_000
   expect new_lp == 1_000_000_000
   expect simple == 1
   expect strategies == 0
+  expect withdrawal_only == False
```

Comment:
I think this should suffice to test that we are checking for withdraws correctly, since it shows that the flag changes from True to False

Reply:
yes, just need to add a withdrawal only test case